src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+    static final int COUNT_BALLS = 3;
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        final Ball[] magicBalls = new Ball[COUNT_BALLS];
+        for (int i = 0; i < magicBalls.length; i++) {
+            magicBalls[i] = lottery.getRandomBall();
+            System.out.println(magicBalls[i]);
+        }
```

Comment:
It right solution, but to simplify it you can remove `magicBalls` array

```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < COUNT_BALLS; i++) {
            System.out.println(lottery.getRandomBall());
        }
```